modules/workaround-make.nix

```diff
@@ -59,6 +59,11 @@ in
   ###### implementation
 
   config = {
+    warnings = lib.optionals cfg.enable [
+      ("system.workaround.make-posix-spawn.enable " +
+       "should no longer be needed on 21.05")
```

Comment:
Maybe clarify that this option will be removed in 21.11 and if anyone still needs this, they should create an issue?

Reply:
expanded the warning text